.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,29 @@
+name: lint_python
+on: [pull_request, push]
```

Comment:
I don't think we need the push. If something has been approved and merged, it's because it passes tests. And we won't push to master directly. Am I understanding this correctly?

Reply:
If push is not included the no test is run for the contributor so let’s only run on push.

Reply:
can you elaborate more your answer? we want the GHA to be checks in the PRs, and we don't need them when we push to master

Reply:
Pull Request only happens on the main repo end of things.
Push happens both on the contributor's repo and on the main repo.
This means that if only `pull_request` is set then no tests are run when a contributor pushes to a new branch.

Reply:
fair enough. Would `push` allow to have the tests visible in the PR?